socorro/unittest/external/boto/test_crashstorage.py

```diff
@@ -193,126 +133,82 @@ def _fake_unredacted_processed_crash_as_string(self):
         s = json.dumps(d)
         return s
 
-    def test_save_raw_crash_1(self):
-        boto_s3_store = self.setup_mocked_s3_storage()
+    @mock_s3_deprecated
```

Comment:
This file might be easier to read through by just looking at the file and not what changed.